src/main/java/org/kpmp/participant/ParticipantRepoDataTypeInformation.java

```diff
@@ -0,0 +1,45 @@
+package org.kpmp.participant;
+import org.kpmp.dataSummary.AtlasRepoSummaryLinkInformation;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ParticipantRepoDataTypeInformation {
+    
+	private String dataType;
+	private Integer count;
+    private AtlasRepoSummaryLinkInformation linkInformation;
+
+	public ParticipantRepoDataTypeInformation() {
```

Comment:
If you don't mark this as a component, Spring will not complain. Since this object is not getting autowired into any class, it does not need to be marked as a component.

So, let's remove the @Component and the default constructor.